README.md

```diff
@@ -44,17 +44,24 @@ Within each section, different configuration variables could be defined. In our
 # Module automatic import
 
 Each python file in services/ folder with defined class name that is derived from BaseService is imported as a module to SparcClient class.
-For example, pennsieve.py file 
+
+For example, Pennsieve module could be used in the following way: 
 
 ```python
 from sparc.client import SparcClient
-a = SparcClient(connect=False, config_file='config.ini')
-module = a.pennsieve.connect()
-module.info() #execute internal functions of the module
+client = SparcClient(connect=False, config_file='config/config.ini')
 
-# alternatively connect all the services available
-a.connect()  #connect to all services
+# Run module prerequisities, e.g. start Pennsieve agent in the background
+!pennsieve agent
```

Comment:
<img width="182" alt="image" src="https://github.com/nih-sparc/sparc.client/assets/778048/b4afddea-af58-4bea-ad71-69c670eb8c2f">
Is that a special character?

Reply:
No, this command calls console in order to execute command that launches pennsieve agent.
More details:
https://stackoverflow.com/questions/44604027/running-interactive-command-line-code-from-jupyter-notebook